### PR TITLE
ADEN-9053 adding changes from gamepedia-ads

### PR DESCRIPTION
**Pull request by preludek**

### CODE REVIEW

**Comment by dmnsobczak** [comment]
Previous PR in `gamepedia-ads`: https://github.com/Wikia/gamepedia-ads/pull/79

File: platforms/gamepedia/slots.ts [hunk 1]
```diff
@@ -10,6 +10,21 @@ class SlotsContext {
 		});
 	}
 
+	setupSlotVideoAdUnit(adSlot: any, params: any): void {
```
Comment (dmnsobczak) [inline platforms/gamepedia/slots.ts]: Some types? This `any` is ugly :<

File: platforms/gamepedia/slots.ts [hunk 2]
```diff
@@ -93,12 +122,13 @@ class SlotsContext {
 		this.setState('cdm-zone-03', true);
 		this.setState('cdm-zone-04', !context.get('state.isMobile'));
 		this.setState('cdm-zone-06', true);
+		this.setState('incontent_player', true, true);
```
Comment (dmnsobczak) [inline platforms/gamepedia/slots.ts]: wgVariable here?

Comment (preludek) [reply platforms/gamepedia/slots.ts]: done

File: platforms/gamepedia/slots.ts (hunk 1 above)
Comment (fraszczakszymon) [reply platforms/gamepedia/slots.ts]: +1

File: platforms/gamepedia/slots.ts [hunk 3]
```diff
@@ -84,6 +99,20 @@ class SlotsContext {
 					rv: 1,
 				},
 			},
+			incontent_player: {
+				avoidConflictWith: '.repeatable-boxad',
```
Comment (fraszczakszymon) [inline platforms/gamepedia/slots.ts]: Not needed

File: platforms/gamepedia/slots.ts (hunk 1 above)
Comment (preludek) [reply platforms/gamepedia/slots.ts]: done

File: platforms/gamepedia/slots.ts (hunk 3 above)
Comment (preludek) [reply platforms/gamepedia/slots.ts]: done

File: platforms/gamepedia/slots.ts [hunk 4]
```diff
@@ -93,12 +128,13 @@ class SlotsContext {
 		this.setState('cdm-zone-03', true);
 		this.setState('cdm-zone-04', !context.get('state.isMobile'));
 		this.setState('cdm-zone-06', true);
+		this.setState('incontent_player', context.get('options.video.isOutstreamEnabled'), true);
 	}
 
-	private setState(slotName: string, state: boolean): void {
+	private setState(slotName: string, state: boolean, skipCheck: boolean = false): void {
```
Comment (fraszczakszymon) [inline platforms/gamepedia/slots.ts]: It isn't needed, am I right? It adds a lot of logic to a simple `setState` method.

Comment (preludek) [reply platforms/gamepedia/slots.ts]: deleted this logic

**Pull request merged**